ext/standard/stubs/UUIDParsingException.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * PHP stub with full documentation for the native UUIDParsingException class.
+ *
+ * This class is available as polyfill for PHP 5 and 7 at
+ * [GitHub](https://github.com/Fleshgrinder/php-uuid).
+ */
+
+/**
+ * Thrown if parsing of a UUID from a string fails.
+ *
+ * ## Examples
+ * ```
+ * <?php
+ *
+ * try {
+ *     UUID::parse('php');
+ * }
+ * catch (UUIDParsingException $e) {
+ *     echo $e->getMessage();  // Expected at least 32 characters, but got 3 characters
+ *     echo $e->getInput();    // php
+ *     echo $e->getPosition(); // 0
+ * }
+ *
+ * try {
+ *     UUID::parse('12345678-1234-1234-1234-123456789php');
+ * }
+ * catch (UUIDParsingException $e) {
+ *     echo $e->getMessage();  // Expected hexadecimal digit, but found 'p' (0x70)
+ *     echo $e->getInput();    // 12345678-1234-1234-1234-123456789php
+ *     echo $e->getPosition(); // 33
+ * }
+ *
+ * try {
+ *     UUID::parse('12345678-1234-1234-1234-123456789abcdef');
+ * }
+ * catch (UUIDParsingException $e) {
+ *     echo $e->getMessage();  // Expected no more than 32 hexadecimal digits
+ *     echo $e->getInput();    // 12345678-1234-1234-1234-123456789abcdef
+ *     echo $e->getPosition(); // 37
+ * }
+ *
+ * ?>
+ * ```
+ *
+ * @see \UUID::parse()
+ */
+final class UUIDParsingException extends Exception {
```

Comment:
I think `UUIDParseException` name would be more appropriate. Looking for other similar examples [ParseError](http://php.net/manual/en/class.parseerror.php) could be good one:
> ParseError is thrown when an error occurs while parsing PHP code